### DISCUSSION TITLE
more prompt scheme documentation?

### DISCUSSION THREAD

**Discussion by pan-genome** [evo-design/evo#67] [Q&A]
could you provide more prompt scheme documentation beyond the one in the example colab notebook using the "greengenes-style lineage strings"? are there other possible natural language-like prompts? or even zero-shot prompting possibility?

**Comment by brianhie** [comment]
The only special "natural language like" prompts are the lineage strings for the 131k model only.

You can take a look at [bac120_taxonomy.tsv](https://data.gtdb.ecogenomic.org/releases/latest/bac120_taxonomy.tsv) in GTDB for a list of strings.

Some example species prompts to get started are
```
|d__Bacteria;p__Pseudomonadota;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Escherichia;s__Escherichia||
|d__Bacteria;p__Pseudomonadota;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Klebsiella;s__Klebsiella pneumoniae||
|d__Bacteria;p__Bacillota;c__Bacilli;o__Staphylococcales;f__Staphylococcaceae;g__Staphylococcus;s__Staphylococcus aureus||
|d__Bacteria;p__Tenericutes;c__Mollicutes;o__Mycoplasmatales;f__Mycoplasmataceae;g__Mycoplasma;s__Mycoplasma genitalium||
```

**Comment by mfdj2002** [comment]
Hi I'm still kind of confused. Are you saying that the 131k model recognizes the lineage strings of all bacteria you can find in GTDB? Also wondering are these special prompts supposed to work on the together web ui provided? I tried "|d__Bacteria;p__Tenericutes;c__Mollicutes;o__Mycoplasmatales;f__Mycoplasmataceae;g__Mycoplasma;s__Mycoplasma genitalium||" and it returned Error. Thank you in advance!
